app/representers/api/image_representer.rb

```diff
@@ -17,19 +17,26 @@ def self_url(represented)
   end
 
   property :id, writeable: false
-  property :filename, writeable: false
+  property :filename
```

Comment:
This was part of my initial pass, where CMS itself (carrierwave) was doing the resize/thumbnails.  Mostly it was useful in dev to leave S3/Lambdas out.

But now that we've moved away from that, I can see removing direct-PUT-ing the file.  Should also update the Dockerfile to not include imagemagick, then.